### ISSUE TITLE
Pre-train ViT on unsupervised task?

### ISSUE THREAD

**Issue by neel04** [yitu-opensource/T2T-ViT#43]
Thanks a lot for this repo! I am looking to pre-train `ViT` in an unsupervised way, but it doesn't seem like it's possible with this repo.

Is there any way I could do that? would you happen to know any other method/repo where we can pre-train it in an unsupervised manner (i.e on a dataset not labeled)?

do you think that specifying a dummy class and pre-training that way may provide extra accuracy when fine-tuning on the labeled dataset?

Many thanks :hugs:

**Comment by yuanli2333** [comment]
Hi,  if you want to know more about self-supervised learning by vision transformer, I recommend this [paper](https://arxiv.org/abs/2104.14294)
